grails-app/controllers/org/pih/warehouse/reporting/ConsumptionController.groovy

```diff
@@ -295,7 +295,9 @@ class ConsumptionController {
             }
 
             def csv = dataService.generateCsv(csvrows)
-            response.setHeader("Content-disposition", "attachment; filename=\"Consumption-${new Date().format("dd MMM yyyy hhmmss")}.csv\"")
+            response.setHeader("Content-disposition", "attachment; filename=\"Consumption-" +
+                    "${command.fromLocations.size() > 1 ? command.fromLocations : command.fromLocations.first()}" +
```

Comment:
How this looks like if fromLocations size is bigger than one? How it is printed?

Reply:
@awalkowiak When there is more than one location chosen they will be printed in brackets. So it would look like this : Consumption-[FirstLocation, SecondLocation, ...]-Date

Reply:
ok, but if fromLocations.size() will be == 0, then this part command.fromLocations.first() will produce error...